src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConstructorSnippetProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Snippets.SnippetProviders
+{
+    internal abstract class AbstractConstructorSnippetProvider : AbstractSnippetProvider
+    {
+        public override string SnippetIdentifier => "ctor";
```

Comment:
Probaably want to update [s_builtInSnippetsWithoutReplacement](https://github.com/dotnet/roslyn/blob/main/src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs#L34) as well